src/app/helpers/announcement/controller.js

```diff
@@ -214,7 +216,7 @@ class AnnouncementController {
         }
       }
 
-      this.objectStoreRest.findObject(query)
+      this.objectStoreRest.findObject(query, true)
```

Comment:
Don't pass boolean directly. Should create a variable and pass that. This helps with maintainability.